### PR TITLE
Unified run function

### PR DESCRIPTION
**Pull request by moshebeutel**
@mhallak _run_logical_sim_ now written for both _logic_sim_ and for real simulation  - depends on _is_logical_ argument
_logic_sim_  - Chalutzi scenario is tested
real simulation - not really tested of course...
 I keep _run_scenario_ function for refernce

### CODE REVIEW

File: v0_train_planner.py [hunk 1]
```diff
@@ -216,342 +213,146 @@ def order_drones_look_at(actions, suicide_drone, sensor_drone):
...
-        #     elif drn_state.is_scanner3:  ## Shouldn't happen
-        #         logging.info('Shouldnt happen')
-        #         if dist3d(log_drn.pos, at_scanner3) <= min_dist:
-        #             add_action(action_list, 'MOVE_TO', 'SensorDrone', (at_scanner1,))
-        #             add_action(action_list, 'LOOK_AT', 'SensorDrone', (at_house1,))
-        #             drn_state.phase5()
+        # DONE LOGIC
+        if done or step > logic_sim.MAX_STEPS :
+            reason = "Logical Simulation" if done else "step is " + step._str__()
+            num_of_dead += len([ff for ff in log_enemies if ff.health == 0.0])
```
Comment (mhallak) [inline v0_train_planner.py]: log_enemies?

File: v0_train_planner.py [hunk 2]
```diff
@@ -216,342 +213,146 @@ def order_drones_look_at(actions, suicide_drone, sensor_drone):
...
+            num_of_lost_devices += len([i for i in log_entities if log_entities[i].health == 0.0])
+            done = True
```
Comment (mhallak) [inline v0_train_planner.py:283]: log_entities?

**Comment by mhallak** [comment]
There is a problem in lines of v0_planner: 280 and 281 - The method doesn't know logical entities.

File: v0_train_planner.py (hunk 1 above)
Comment (moshebeutel) [reply v0_train_planner.py]: @mhallak bug fixed

**Pull request merged**